akka-http-core/src/test/resources/routing-settings-test.conf

```diff
@@ -0,0 +1,9 @@
+akka.http.routing {
+  verbose-error-messages = off
+  file-get-conditional = on
+  render-vanity-footer = yes
+  range-coalescing-threshold = 80
+  range-count-limit = 16
+  decode-max-bytes-per-chunk = 1m
+  file-io-dispatcher = "test-only"
+}
```

Comment:
I'd prefer this config in-line in the tests, it's not something we're using anywhere else